pyproject.toml

```
@@ -0,0 +1,59 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pytom-template-matching-gpu"
+version = "0.3.4"
+description = "GPU template matching from PyTOM as a lightweight pip package"
+readme = "README.md"
+license = {file = "LICENSE"}
+authors = [
+    {name = "Marten Chaillet"},
```

```suggestion
    {name = "Marten Chaillet", email = "m.l.chaillet@uu.nl"},
```
For now we can still add my uu mail as well

---

fair enough